### DISCUSSION TITLE
输出到kafka验证不了tls

### DISCUSSION THREAD

**Discussion by q-cloud-ll** [fluent/fluentd#4733] [Q&A]
### What is a problem?

我在日志里面看到报如下错误：Send exception occurred: SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed (ok)
我的kafka版本是0.10.2，我在其他进程中是可以连上kafka的，但是在fluentd会报tls错误，如何解决

### Environment

```markdown
- Fluentd version:
- TD Agent version:
- Fluent Package version:
- Docker image (tag):
- Operating system:
- Kernel version:
```

**Comment by cosmo0920** [comment]
请写下您的使用配置。如果没看得您的使用的配置和Fluentd的版本，我们就不知道什么错的地方。
